packages/aeproject-cli/aeproject-contracts/aeproject-contracts.js

```diff
@@ -7,18 +7,36 @@ const {
     spawn
 } = require('promisify-child-process');
 
+const prompt = require('aeproject-utils').prompt;
+
 const contractsConstants = require('./contracts-constants.json');
 
 let nodeModulesPath;
 let contractAeppProjectPath;
 
 const run = async (options) => {
+    await installYarn();
     await mainFlow(options);
 };
 
+const installYarn = async () => {
+    let isYarnInstalled = false;
+    try {
+        await exec('yarn --version');
+        isYarnInstalled = true;
+    } catch (error) {
+        // yarn not found...
+    }
+
+    if (!isYarnInstalled) {
+        await prompt('Yarn not found! Contracts depends on it. Do you want to install "yarn"?', exec, 'brew install yarn'); // brew install yarn
```

Comment:
brew is a third party dependency and not sure if it would work on windows. Can we user npm instead?

Reply:
yes, we can